schema.graphql

```diff
@@ -34,28 +34,26 @@ type GraphNetwork @entity {
   isPaused: Boolean!
   "True if the protocol is partially paused"
   isPartialPaused: Boolean!
-  "Governor of the controller i.e. the whole protocol"
+  "Governor of the controller (i.e. the whole protocol)"
   governor: Bytes!
   "Pause guardian address"
   pauseGuardian: Bytes!
 
   # Staking global parameters
-  "Percentage of fees going to curators"
+  "Percentage of fees going to curators. In parts per million"
   curationPercentage: Int!
-  "Percentage of fees burn as protocol fee"
+  "Percentage of fees burn as protocol fee. In parts per million"
   protocolFeePercentage: Int!
   "Ratio of max staked delegation tokens to indexers stake that earns rewards"
   delegationRatio: Int!
-  "Period to wait before fees can be claimed in rebate pool"
+  "Epochs to wait before fees can be claimed in rebate pool"
   channelDisputeEpochs: Int!
-  "Period to wait before delegators can settle"
+  "Epochs to wait before delegators can settle"
   maxAllocationEpochs: Int!
   "Time in blocks needed to wait to unstake"
   thawingPeriod: Int!
   "Minimum time an Indexer must use for resetting their Delegation parameters"
   delegationParametersCooldown: Int!
-  "The number of times indexing rewards are handed out each epoch"
-  indexingRewardsPerEpoch: Int!
```

Comment:
This field is very old and no longer in use, it represents an old way the rewards were implemented